### PR TITLE
Change asymmetric visibility example to show how it simplifies code

### PR DESCRIPTION
**Pull request by theodorejb**
The previous example wasn't very helpful, since it only showed invalid code that will error (and could just as easily be replaced with a `readonly` property).

### CODE REVIEW

**Comment by saundefined** [comment]
LGTM, thanks!

@TimWolla can you also take a look?

File: releases/8.4/release.inc [hunk 1]
```diff
@@ -178,11 +185,13 @@ PHP
...
     public private(set) string $version = '8.4';
-}
 
-$phpVersion = new PhpVersion();
-var_dump($phpVersion->version); // string(3) "8.4"
-$phpVersion->version = 'PHP 8.3'; // Visibility error
+    public function increment(): void
+    {
+        [$major, $minor] = explode('.', $this->version);
+        $this->version = "$major." . (intval($minor) + 1);
```
Comment (TimWolla) [inline releases/8.4/release.inc:192]: Just on my phone and can't make an actual suggestion, but would make $minor++ as a separate statement to keep the complexity of this line low and a simple interpolation.

**Review by TimWolla — COMMENTED** [review]
Example is clearly an improvement.

File: releases/8.4/release.inc (hunk 1 above)
Comment (theodorejb) [reply releases/8.4/release.inc]: Updated!

**Review by saundefined — APPROVED** [review]

**Pull request merged**